### DISCUSSION TITLE
Croatian Chanell's

### DISCUSSION THREAD

**Discussion by TheLegnar** [iptv-org/community#49] [Q&A] [answered]
hey guys im really new here .
and new to iptv ....
So im not shure what is going on , on https://iptv-org.github.io/?q=%22croatia%22  say's correct amount of channels 
but on the list from playlist we get only 10 of them ?

anyone know why ?
and can i do some stuff myself to add those channels to the list and if so i might need a guide on how do i to it myself ?
or maybe anyone can help outh?

and why is there no ex yugoslavia channels as well? in categories ?

**Comment by BellezaEmporium** [comment, marked as answer]
I should add this answer to the FAQ since it's asked all the time...

@sguinetti we'll need to add this one up.

What you see in the IPTV GH website is a list of the channels we have in [**the database**](https://github.com/iptv-org/database/blob/master/data/channels.csv) and does not reflect the [**streams**](https://github.com/iptv-org/iptv/tree/master/streams) we have in this repository. 

The database simply aims to reference all known channels to this day (which is why we have 19k channels).
This playlist contains over 7k channels.
Please note we'll never get all the channels to our playlists for various and very obvious reasons : DMCA blocks for certain paid channels, channels being satellite only etc etc...

**Comment by sguinetti** [comment]
I think "country:HR". Like this: https://iptv-org.github.io/?q=country:HR 

Thanks. I will update the FAQ.
